src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/CommonLogger.cs

```diff
@@ -11,12 +13,14 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
     /// </summary>
     public class CommonLogger : Common.ILogger
     {
-        public CommonLogger(ILogger logger)
+        private static EventId logEvent = new EventId(23847);
```

Comment:
Either const-ize 23847 with a well-named field or add a comment explaining where the magical number came from.